easybuild/easyblocks/o/openfoam.py

```diff
@@ -282,15 +307,16 @@ def sanity_check_step(self):
                                                     "refineMesh", "vorticity"]]
         # check for the Pstream and scotchDecomp libraries, there must be a dummy one and an mpi one
         if 'extend' in self.name.lower():
-            libs = [os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]] + \
-                   [os.path.join(libsdir, "libscotchDecomp.so")]
+            libs = [os.path.join(libsdir, "libscotchDecomp.so")]
+            if LooseVersion(self.version) < LooseVersion('3.2'):
+                libs.extend([os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]])
```

Comment:
The Pstream library has apparently been removed in this commit:
http://sourceforge.net/p/foam-extend/foam-extend-3.2/ci/4c23c3f9e812674bf54b6a26e58f32511bb6af69/

It looks like it is now integrated in the main foam library.

What is important to check is that OpenFoam executables and libraries (eg libfoam.so) are actually linked to the MPI library of the toolchain, and not the one from the ThirdParty directory that they provide.
Running `ldd lib/linux64GccDPOpt/libfoam.so` should help to check this.